css/index.css

```diff
@@ -1,25 +1,43 @@
 .textInput {
   width: 100%;
-  padding: 12px 20px;
-  margin: 8px 0;
-  display: inline-flex;
-  border: 1px solid white;
-  border-radius: 4px;
-  box-sizing: border-box;
-  color: white;
-  background-color: black;
+  padding: 12px 20px !important;
```

Comment:
dont use the important flag. If youre still having problems afterwards with the css not applying, create a new classname for this css element, and use that.

Reply:
fixed